app/src/main/java/edu/mills/cs115/fruitthief/database/PopulateFruitTable.kt

```diff
@@ -0,0 +1,37 @@
+package edu.mills.cs115.fruitthief.database
+
+import kotlinx.coroutines.runBlocking
+import timber.log.Timber
+
+class PopulateFruitTable(fruitTreeDao: FruitTreeDAO) {
+
+    init {
+        runBlocking{
+            if (fruitTreeDao.getFruitList().isEmpty()) {
+                Timber.i("Populating table")
+                fruitTreeDao.insert(Fruit(0, "Apple", "HIJK"))
```

Comment:
There's no reason to always include 0. Make it a default value for the parameter.